setup.py

```diff
@@ -41,9 +41,9 @@
         "pandas>=0.25.0,<0.26.0",
         "tqdm>=4.33.0,<5.0.0",
         "scikit-learn>=0.20.2,<0.22.0",
-        "torch>=1.1.0,<1.2.0",
+        "torch>=1.2.0,<2.0.0",
+        "tensorboard>=1.14.0,<2.0.0",
```

Comment:
how did we decide on this range?

Reply:
I got an import error that looked like this:
```
ImportError: TensorBoard logging requires TensorBoard with Python summary writer installed. This should be available in 1.14 or above
```